src/simplify_qdq.cpp

```diff
@@ -0,0 +1,156 @@
+#include <migraphx/simplify_qdq.hpp>
+#include <migraphx/instruction.hpp>
+#include <migraphx/iterator_for.hpp>
+#include <migraphx/make_op.hpp>
+#include <migraphx/program.hpp>
+#include <migraphx/shape.hpp>
+#include <migraphx/matcher.hpp>
+#include <migraphx/dead_code_elimination.hpp>
+#include <migraphx/pass_manager.hpp>
+#include <migraphx/op/convolution.hpp>
+#include <migraphx/op/quant_convolution.hpp>
+#include <migraphx/op/dot.hpp>
+#include <migraphx/op/quant_dot.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+
+std::unordered_set<std::string> get_quantizable_op_names()
+{
+    static std::unordered_set<std::string> s = {"convolution", "dot"};
+    return s;
+}
+
+struct match_find_add_bias
+{
+    auto matcher() const
+    {
+        return match::name("dequantizelinear")(match::arg(0)(
```

Comment:
This assumes there is always an "add" after "convolution", and after the "add" there is a QDQ pair. This is not true. (Maybe the mobilenetv2 is the case), for example, in the Alexnet (I shared you the model), one scenario is like: 

```
main:@128 = multibroadcast[output_lens={1, 64, 27, 27}](main:@63) -> float_type, {1, 64, 27, 27}, {0, 0, 0, 0}
main:@129 = multibroadcast[output_lens={1, 64, 27, 27}](main:@64) -> int8_type, {1, 64, 27, 27}, {0, 0, 0, 0}
main:@130 = quantizelinear(main:@127,main:@128,main:@129) -> int8_type, {1, 64, 27, 27}, {46656, 729, 27, 1}
main:@131 = multibroadcast[output_lens={1, 64, 27, 27}](main:@63) -> float_type, {1, 64, 27, 27}, {0, 0, 0, 0}
main:@132 = multibroadcast[output_lens={1, 64, 27, 27}](main:@64) -> int8_type, {1, 64, 27, 27}, {0, 0, 0, 0}
main:@133 = dequantizelinear(main:@130,main:@131,main:@132) -> float_type, {1, 64, 27, 27}, {46656, 729, 27, 1}
main:@134 = convolution[padding={2, 2, 2, 2},stride={1, 1},dilation={1, 1},group=1,padding_mode=0](main:@133,main:@102) -> float_type, {1, 192, 27, 27}, {139968, 729, 27, 1}
main:@135 = broadcast[axis=1,dims={1, 192, 27, 27}](main:@105) -> float_type, {1, 192, 27, 27}, {0, 1, 0, 0}
main:@136 = add(main:@134,main:@135) -> float_type, {1, 192, 27, 27}, {139968, 729, 27, 1}
main:@137 = pooling[mode=max,padding={0, 0, 0, 0},stride={2, 2},lengths={3, 3},ceil_mode=0](main:@136) -> float_type, {1, 192, 13, 13}, {32448, 169, 13, 1}
```

After the convolution, there is no QDQ.